raiden-dapp/src/components/RaidenDialog.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <blurred-overlay :show="visible">
+  <blurred-overlay :show="visible" :fullscreen="true">
```

Comment:
Since this is not attached to a variable you can do `fullscreen` instead of `:fullscreen="true"` 

That is the reason why the property has `type: Boolean` in the blurred-overlay component